modules/apps/commerce/commerce-service/service.xml

```diff
@@ -357,6 +357,7 @@
 		<column name="json" type="String" />
 		<column name="manuallyAdjusted" type="boolean" />
 		<column name="discountManuallyAdjusted" type="boolean" />
+		<column name="priceManuallyAdjusted" type="boolean" />
```

Comment:
I think this should be alphabetically sorted, also the discountManuallyAdjusted property, I'm sorry I didn't catch it before